examples/sops/local-configs/function.yaml

```diff
@@ -1,10 +1,33 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: my-config
+  name: my-decrypt-config
   annotations:
     config.k8s.io/function: |
       container:
         image: gcr.io/kpt-functions/sops
+        envs:
+        - SOPS_IMPORT_PGP
     config.kubernetes.io/local-config: "true"
 data:
+  cmd: 'decrypt'
+  cmd-json-path-filter: '$[?(@.metadata.name=="toDecrypt")]'
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-encrypt-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image: gcr.io/kpt-functions/sops
+        envs:
+        - SOPS_IMPORT_PGP
+        #- SOPS_PGP_FP
```

Comment:
Why is this commented?

Reply:
there are 2 supported options how to provide this parameter:
1. via function parameter pgp, e.g.:
pgp: 'FBC7B9E2A4F9289AC0C1D4843D16CEE4A27381B4'

2. or via SOPS_PGP_FP env variable.

I commented env variable because I used the first option.